ckanext/geocat/utils/csw_mapping.py

```diff
@@ -414,3 +448,29 @@ def _map_dataset_rights(node, terms_of_use, default_rights):
                         if ogdch_rights:
                             return ogdch_rights
     return default_rights
+
+
+def get_license_uri_by_name(vocabulary_name):
+    license_vocabulary = get_license_values()
+    for key, value in license_vocabulary.items():
+        if unicode(vocabulary_name) == unicode(value):
+            return key
+    return None
+
+
+def get_license_values():
+    g = Graph()
+    license_mapping = {}
+    for prefix, namespace in license_namespaces.items():
+        g.bind(prefix, namespace)
+    file = os.path.join(__location__, 'license.ttl')
+    g.parse(file, format='turtle')
+    for ogdch_license_ref in g.subjects(predicate=RDF.type,
+                                        object=SKOS.Concept):
+        license_mapping[ogdch_license_ref] = None
+        for license_pref_label in g.objects(subject=ogdch_license_ref,
+                                            predicate=SKOSXL.prefLabel):
+            for license_literal in g.objects(subject=license_pref_label,
+                                             predicate=SKOSXL.literalForm):
+                license_mapping[ogdch_license_ref] = license_literal
+    return license_mapping
```

Comment:
Can we move these functions into `ogdch_map_utils.py` please?

Reply:
Do not need these functions anymore, deleted them, because it is already checked against the values of the geocat-term-of-use.xml, what is the same, without the cc license. We decided to take the default avlue of the harvester, if its a cc license.